src/quatrex/core/scba.py

```diff
@@ -197,16 +202,26 @@ def __init__(
             if os.path.isfile(energies_path):
                 self.coulomb_screening_energies = np.load(energies_path)
             else:
-                self.coulomb_screening_energies = self.electron_energies
-
-            self.p_coulomb_screening = PCoulombScreening(...)
+                self.coulomb_screening_energies = -(
+                    self.electron_energies - self.electron_energies[0]
+                )[::-1]
+                # Remove the zero energy to avoid division by zero.
+                self.coulomb_screening_energies[-1] = -1e-6
```

Comment:
When we slice the Polarization after doing the convolution we select the first half of the energies, which correspond to energies going from -(E_max-E_min) to zero 0, so that is why the energies are negated and reversed. This energy vector is only interesting when the OBC's are calculated from the fluctuation-dissipation theorem, which I haven't seen working